src/app/app.routes.js

```diff
@@ -142,7 +142,19 @@ angular.module('librarium')
           views: {
             'main@': {
               templateUrl: '../app/components/users/views/show.html',
-              controller: 'TemporaryIdController as ctrlUserId'
+              controller: 'UsersShowController as uShowCtrl'
+            },
+            'favoriteBooks@root.user': {
+              templateUrl: '../app/components/users/views/favorite_books.html',
+              controller: 'FavoriteBooksController as fBooksCtrl'
```

Comment:
You can have a separate controller for each embedded view or use the controller of the parent view. It all depends on how complex is the controller